components/automate-ui/src/app/pages/signin/signin.component.ts

```diff
@@ -48,8 +48,6 @@ export class SigninComponent implements OnInit {
   }
 
   setSession(): void {
-    const isLocalUser = this.id.federated_claims &&
-      this.id.federated_claims.connector_id === 'local';
```

Comment:
Consolidation of this in the next two files is unrelated, rather a preventative measure that might have caused problems in the future.